oblib/taxonomy_misc.py

```diff
@@ -283,7 +283,7 @@ def __init__(self):
         self._documentation = self._load_documentation()
 
     def _load_documentation(self):
-        label_file = "solar_2018-03-31_r01_lab.xml"
+        label_file = "solar_2019-02-27_r01_lab.xml"
```

Comment:
suggest that we search for a file matching pattern '*lab.xml' and if we find only one, we import that as the label_file.  Otherwise oblib will be specific to one taxonomy release.

There's a nice example of a utility function to find files [here](https://stackoverflow.com/questions/1724693/find-a-file-in-python)

Reply:
I think we need to add multiple version support to the roadmap and get rid of all of the hard-coded filenames.  I don't know what release (1.1.0, 1.2.0, etc...) this should be added at.  One question that has always been on the table is how will the naming convention work between releases and I would like to reach a definitive agreement with the solar-taxonomy/XBRL team before changing this part of the code.

Reply:
I agree.  I think, until we can set up something like SOLAR_TAXONOMY_DIR for the label and elements file, that searching for the pattern is better than having the fixed name.